views/pages/ParticipantsPage.jsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react'
-import { Box } from '@mui/material'
+import { Box, Typography, useMediaQuery } from '@mui/material'
 
 import ParticipantsTable from '../tables/ParticipantsTable'
 import ParticipantsSearchForm from '../forms/ParticipantsSearchForm'
 import PageHeader from '../components/PageHeader'
 import useParticipantsList from '../hooks/useParticipantsList'
 
 const ParticipantsPage = () => {
+  const isMidOrSmallerDisplay = useMediaQuery('(max-width:1100px)')
```

Comment:
you can use media queries directly in the sx prop, so I don't think we need to use this hook. For example, you could have something like this contrived example:

```tsx
<Box
  sx={{
    display: 'flex',
    '@media (max-width: 1100px)': {
      display: 'none',
    }
  }}
/>
```

Reply:
looks like you can also use the built-in breakpoints, so the above example could be

```tsx
<Box
  sx={{
    display: {
      xxs: 'flex',
      lg: 'none',
    },
  }}
/>

Reply:
[a48e441](https://github.com/AMP-SCZ/dpdash/pull/561/commits/a48e4417d763b2a44e3234ae6e9db27931d3ddc6)